packages/ui/v2/core/Shell.tsx

```diff
@@ -780,9 +781,10 @@ function SideBar() {
           <Navigation />
         </div>
 
-        {isCalcom && <Tips />}
-
-        <div className="fixed left-1 bottom-0 w-4 bg-gray-50 before:absolute before:left-0 before:-top-20 before:h-20 before:w-48 before:bg-gradient-to-t before:from-gray-50 before:to-transparent lg:left-4 lg:w-48">
```

Comment:
@JeroenReumkens this was a neat idea but caused a lot of headaches with the `<Tips>` component

Reply:
Ah damn... Totally forgot about that component 😢 Perhaps the tips could be nested inside the fixed wrapper. Want me to try this still?